course/resources/rqm-ecologized.md

```diff
@@ -0,0 +1,227 @@
+- Academic publishing
+    - So, how do you decide where to publish? It’s easy to find resources about choosing a journal from a more conventional framework (TL:DR- most of them are some variation of “Choose the journal with the highest impact factor that also fits your work”)- but issues around open access muddies this old advice a bit. This is a really good opportunity to talk about issues with valuing science and scientists by impact factors of journals. We suggest reading High-impact open access journals and Eigenfactor Index of Open Access Fees to get you started. Open access seems simple on the surface, but we get into a very tangled web of corporate interests, biased metrics, and misinformation very quickly.
+- Participatory models for bigger science
+    - With increased connectivity, we have the opportunity to bring our work to audiences that have never been engaged in science before. This is a good opportunity for students to explore the differences in participatory models between fields, and discuss what engagement means in different fields. We provide a more extreme example, with participatory biomedical research- how do we engage people in our work when the direct ties to their well-being are more tenuous? Next week we'll look at participatory models in ecology.
+- Check up on the collaborative process
+    - An informal discussion that doubles as a chance to flesh out where the project is going. How is the collaborative process working? Do we need to make adjustments to get the project done? Use our answers to these questions to help populate/adjust the github issue list.
+- Looking towards the future
+    - Use this discussion as an opportunity to get feedback on the course, and to help guide students in embracing open science and reproducible research in their future endeavours. 
+
+
+### Exercises
+These are largely covered by the topics in the lectures, but if you see something that is not and that you think is important, comment on that line suggesting where to fit it in.
+
+- Find university IP policy, and discuss
+- Sign up for github
+- Brainstorm questions for project data set
+    - What you'd like to ask of your project data set? What questions can be answered with the data set directly? What questions can be answered with additional contextual data? Ask a variety of questions. Write down questions and hypotheses in your blog/etherpad so you have them on record.
+- Selecting collaborative writing tools
+    - Search the web with your group for collaborative writing tools and platforms. Some places to start: Overleaf, Authorea, Google Docs...even a Word file in a shared Dropbox… really, anything that would allow multiple people to have edit access to a document.
```

Comment:
I agree Rmarkdown might be better, then we don't have to setup accounts on different platforms. And it will be easier to relate changes in manuscript to changes in code if it on the same repo.